code/core/src/viewport/README.md

```diff
@@ -11,14 +11,14 @@ Storybook Viewport Addon allows your stories to be displayed in different sizes
 Viewport is part of [essentials](https://storybook.js.org/docs/essentials) and so is installed in all new Storybooks by default. If you need to add it to your Storybook, you can run:
```

Comment:
The whole installation part is not necessary anymore

Reply:
All the merged in packages in the core retain their original `README.md` and haven't been updated to reflect this change.

The long term goal is to have a proper README.md at the core package level, but writing that is a not a good idea, right now, since so much is exposed, that we want to make private or even remove.

Removing the README.md files of the packages, would likely have been annoying since there might still be some useful info in there, especially considering global file search.

Making them accurate is likely not really a good use of our time either.

So we're left with these README.md files in core, that make little sense.

I'm hoping that when we eventually do a project to create a coherent public API, then we'll get rid of all these README.md files.